packit_service/worker/reporting.py

```diff
@@ -61,19 +61,26 @@ def __init__(
         self,
         project: GitProject,
         commit_sha: str,
+        trigger_id: int = None,
         pr_id: Optional[int] = None,
     ):
         logger.debug(
             f"Status reporter will report for {project}, commit={commit_sha}, pr={pr_id}"
         )
         self.project: GitProject = project
         self._project_with_commit: Optional[GitProject] = None
+
         self.commit_sha: str = commit_sha
+        self.trigger_id: int = trigger_id
         self.pr_id: Optional[int] = pr_id
 
     @classmethod
     def get_instance(
-        cls, project: GitProject, commit_sha: str, pr_id: Optional[int] = None
+        cls,
+        project: GitProject,
+        commit_sha: str,
+        trigger_id: int,
+        pr_id: Optional[int] = None,
```

Comment:
I saw some `trigger_id=None`, this may be better and wouldn't make you update code in `Bugzilla` for example:

```suggestion
        pr_id: Optional[int] = None,
        trigger_id: Optional[int] = None,
```

However I see that you are explicit about the arguments, so I leave the choice up to you… _It is_ easier to read with those arguments you have there :shrug: